src/main/java/io/th0rgal/oraxen/events/OraxenFurnitureInteractEvent.java

```diff
@@ -56,9 +57,17 @@ public Player getPlayer() {
         return player;
     }
 
+    /**
+     * @return whether the furniture has barrier
+     */
+    public boolean hasBarrier() {
+        return block != null;
+    }
+
```

Comment:
this should just be able to be checked via the mechanic, like mechanic.hasBarriers()